src/Common/src/System/PasteArguments.cs

```diff
@@ -9,7 +9,7 @@ namespace System
 {
     internal static partial class PasteArguments
     {
-        private static void AppendArgument(StringBuilder stringBuilder, string argument)
+        internal static void AppendArgument(StringBuilder stringBuilder, string argument)
```

Comment:
@AtsushiKan @stephentoub is it okay to convert it to internal ? I did to remove extra allocation of string builder in https://github.com/dotnet/corefx/pull/27191/files#diff-d59c3ebbc96fc8fab97683e7adb5ba0eR448

Reply:
No problem. Is it possible to remove the OS-specific Paste* files from the project now?